examples/get-started/react/widgets/app.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import {createRoot} from 'react-dom/client';
+import DeckGL, {GeoJsonLayer, ArcLayer} from 'deck.gl';
+import {
+  CompassWidget,
+  ZoomWidget,
+  FullscreenWidget,
+  DarkGlassTheme,
+  LightGlassTheme
+} from '@deck.gl/widgets';
+import '@deck.gl/widgets/stylesheet.css';
+
+/* global window */
+const prefersDarkScheme = window.matchMedia('(prefers-color-scheme: dark)');
+const widgetTheme = prefersDarkScheme.matches ? DarkGlassTheme : LightGlassTheme;
+
+// source: Natural Earth http://www.naturalearthdata.com/ via geojson.xyz
+const COUNTRIES =
+  'https://d2ad6b4ur7yvpq.cloudfront.net/naturalearth-3.3.0/ne_50m_admin_0_scale_rank.geojson'; //eslint-disable-line
+const AIR_PORTS =
+  'https://d2ad6b4ur7yvpq.cloudfront.net/naturalearth-3.3.0/ne_10m_airports.geojson';
+
+const INITIAL_VIEW_STATE = {
+  latitude: 51.47,
+  longitude: 0.45,
+  zoom: 4,
+  bearing: 0,
+  pitch: 30
+};
+
+function Root() {
+  const onClick = info => {
+    if (info.object) {
+      // eslint-disable-next-line
+      alert(`${info.object.properties.name} (${info.object.properties.abbrev})`);
+    }
+  };
+
+  return (
+    <DeckGL
+      controller={true}
+      initialViewState={INITIAL_VIEW_STATE}
+      widgets={[
+        new ZoomWidget({style: widgetTheme}),
+        new CompassWidget({style: widgetTheme}),
+        new FullscreenWidget({style: widgetTheme})
+      ]}
+    >
+      <GeoJsonLayer
```

Comment:
Unrelated: should we move get-started to TypeScript? Maybe pure-js can be reserved for JS examples, but react projects are more likely to be using bundlers/plugins/dev frameworks that already support TS.

It's known that JSX layers do not work with TS, though

Reply:
I wonder if the fiber renderer works with TSX? https://github.com/visgl/deck.gl-community/pull/35

Reply:
Is there a value in using JSX layers when the stars of the show, the Widgets, are not expressed as JSX?

Reply:
This got me thinking, if I'm writing a react app and making a custom widget I'd like to know how to write a widget in react. I think [React Portals](https://react.dev/reference/react-dom/createPortal#createportal) offer a pretty seamless interface that we could show in this example